lms/djangoapps/dashboard/tests/test_sysadmin.py

```diff
@@ -437,3 +440,23 @@ def test_user_create_message(self):
             'This email already taken',
             users_api.create_user(**data)
         )
+
+
+@unittest.skipUnless(settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD'),
+                     "ENABLE_SYSADMIN_DASHBOARD not set")
+class TestCourseUsersReport(SysadminBaseTestCase):
+    '''
+    Test creation of course users report
+    '''
+    def test_report_creation_through_request(self):
+        """
+        Test creation of course users report through the post request.
+        """
+        tz_default = timezone.get_default_timezone()
+        tz_before = timezone.get_current_timezone()
+        courses = Mock()
+        course_id = Mock()
+        get_report_data_for_course_users(courses, course_id)
+        tz_after = timezone.get_current_timezone()
+        self.assertEqual(tz_before, tz_default)
+        self.assertEqual(tz_before, tz_after)
```

Comment:
I would suggest separating code blocks with empty lines for a cleaner look and to improve readability, e.g.:
```python
        tz_default = timezone.get_default_timezone()
        tz_before = timezone.get_current_timezone()

        courses = Mock()
        course_id = Mock()

        get_report_data_for_course_users(courses, course_id)

        tz_after = timezone.get_current_timezone()

        self.assertEqual(tz_before, tz_default)
        self.assertEqual(tz_before, tz_after)
```